packages/code-du-travail-frontend/src/outils/indemniteLicenciement/StepNav.js

```diff
@@ -5,18 +5,18 @@ import { theme } from "@cdt/ui";
  * For now, only <Step /> are used but stepItems will
  * allows to navigate between form's steps
  */
-export function StepItems({ activeIndex = 0, items = [] }) {
+export function StepNav({ activeIndex, steps }) {
```

Comment:
For now it's not a step navigation, we don't know if we want to do that. That why it was named StepItems and not StepNavigationItems as it was the case at the start

Reply:
StepItems seemed confusing to me